exercises/isbn-verifier/example.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+function ISBN(isbn) {
+  this.isbn = isbn.replace(/-/g, '');
+
+  this.isValid = function () {
+    if (!this.isbn.match(/^(\d{9}[\dx])$/i)) {
+      return false;
+    }
+
+    var digits = this.isbn.split('');
+    if (digits[9].match(/x/i)) {
```

Comment:
Ignoring case in the regex will allow it to pass for smallcase `x` which is not desired.